docs/Style-Guide.md

```diff
@@ -0,0 +1,180 @@
+
+Condescending languages are words that include:
+
+* Just 
+* Easy
+* Simply
+* Basically
+* Obviously
+
+The reader may not find it easy to use the Fastify's framework and plugins, so avoid words that make it sound simple or easy. Not everyone who reads the documentation has the same level of understanding.
+
+
+### Starting with a verb
+
+Mostly start your description with a verb, which makes it simple and precise for the reader to follow. Use present tense more often, it's easier to read and understand than the past or future tense.
+
+**Example**
+
+ Less like this: There is a need for Nodejs to be installed before you can be able to use fastify.
+
+ More like this: Install Nodejs to make use of fastify.
+
+### Grammatical moods 
+
+Grammatical moods is a great way to express your writing, don't want to sound too bossy while making a direct statement. Know when to switch between indicative, imperative, and Subjunctive moods.
+
+
+**Indicative** - Use when making a factual statement or question. 
+
+Example: Since there is no testing framework available, "fastify recommend ways to write tests".
+
+**Imperative** - Use when giving instructions, actions, commands, or when you write your headings.
+
+Example: Install dependencies before starting application development mode.
+
+
+**Subjunctive** -  Use when making suggestions, hypothesis or non-factual statement.
+
+Example: Reading the documentation on our website is recommended to get comprehensive knowledge on the framework plugin.
+
+### Use **active** voice instead of **passive**
+
+Using active voice is a more compact and direct way of conveying your documentation.
+
+**Example**
+
+Passive: The dependencies of the npm package is installed by nodeJs.
+
+Active: NodeJs installs the npm package dependencies.
+
+## Writing Style
+
+### Documentation titles
+
+When creating a new guide, API or reference in the `/docs/` directory, use short titles that best describe the topic of your documentation. 
+Separate each sentence with the dash *(-)* symbol, which makes your `.md` files clear and readable. It also increases the chance of showing up on search engines (Google, DuckDuckg or Bing) in spheres of Search Engine Optimisation.
+
+**Examples**: <br>
+>`hook-and-plugins.md`, <br> 
+ `adding-test-plugins.md`, <br>
+ `removing-requests.md`.
+
+### Hyperlinks
+
+Hyperlinks should have a clear title of what it references.
+Here is how your hyperlink should look: 
+
+```MD
+<!-- More like this -->
+
+// Add clear & breif description
+[Fastify Plugins] (https://www.fastify.io/docs/latest/Plugins/)
+
+<!--Less like this -->
+
+// incomplete description 
+[Fastify] (https://www.fastify.io/docs/latest/Plugins/)
+
+// Adding title in link brackets
+[](https://www.fastify.io/docs/latest/Plugins/ "fastify plugin")
+
+// Empty title
+[](https://www.fastify.io/docs/latest/Plugins/)
+
+// Adding links localhost URL's instead of using code strings (``)
+[http://localhost:3000/](http://localhost:3000/)
```

Comment:
I'd particularly like to see an example that prohibits "here" links, e.g. `[here](http://example.com)`.

Reply:
I do not understand what you mean.

Was that a suggestion to add "here" links as a good example of hyperlink titles or a bad one, if I'm correct?

Reply:
Definitely under the "bad" category. "here" links are not helpful or descriptive.